### ISSUE TITLE
Timer: show sum of time per day

### ISSUE THREAD

**Issue by kralan** [mtierltd/timetracker#67]
In timer view, a sum of time per activity is calculated and shown. Similarly, a sum of time per day would be convenient.

**Comment by LukeLR** [comment]
Good idea! I just wanted to add that this is already possible under the "Reports" view: Just select a single day in the filter, e.g. "today" or "yesterday", and at the bottom of the table, a sum is displayed. So you can use this feature already without having to wait for it, although a bit more complicated of course.

**Comment by smatthie** [comment]
I support this request.

@LukeLR: You can also get a single table with the total times for all days in the Reports view when you select "group by: none" and "daily". Note that this view always counts days in UTC, so if you're in a different time zone, the result is not what one expects.

**Comment by LukeLR** [comment]
@smatthie Is the UTC bug covered by #26 ?

**Comment by smatthie** [comment]
@LukeLR Yes, I think it is.
